test/functional/change_stream.test.js

```diff
@@ -2277,14 +2321,13 @@ describe('Change Streams', function() {
       });
     });
 
-    // For a ChangeStream under these conditions:
+    // 12. For a ChangeStream under these conditions:
     //   Running against a server <4.0.7.
     //   The batch is empty or has been iterated to the last document.
     // Expected result:
     //   getResumeToken must return the _id of the last document returned if one exists.
-    //   getResumeToken must return startAfter from the initial aggregate if the option was specified.
     //   getResumeToken must return resumeAfter from the initial aggregate if the option was specified.
-    //   If neither the startAfter nor resumeAfter options were specified, the getResumeToken result must be empty.
+    //   If ``resumeAfter`` was not specified, the ``getResumeToken`` result must be empty.
```

Comment:
Same here, why was a reference to `startAfter` removed?

Reply:
Same reason as the other one, it changed in the spec.